### PR TITLE
Pep8 compliance

### PR DESCRIPTION
**Pull request by mfkaptan**
These changes are for files which are in cocotb/cocotb. Changes include mostly:
- Deleting/putting whitespaces.
- Deleting/puttig blank lines.
- Wrapping lines which are >79 chars.
- Silencing some warnings when checking with flake.
- Removing unnecessary 'pass' statements.
This PR is about issue #6 
There are still some unused imports/variables and some other pep8 (and also logical) errors on the code. I left them untouched for now.

### CODE REVIEW

File: cocotb/result.py [hunk 1]
```diff
@@ -28,9 +28,10 @@
 # TODO: Coule use cStringIO?
 import traceback
 import sys
-#from StringIO import StringIO
+# from StringIO import StringIO
```
Comment (chiggs) [inline cocotb/result.py]: We may as well remove this line entirely

File: cocotb/scheduler.py [hunk 2]
```diff
@@ -49,7 +49,7 @@
 # Debug mode controlled by environment variables
 if "COCOTB_ENABLE_PROFILING" in os.environ:
     import cProfile, StringIO, pstats
```
Comment (chiggs) [inline cocotb/scheduler.py]: I'm actually confused as to why import `StringIO` from `io` in `result.py` but import the module directly here.

Assume this is something to do with Python3 compatibility?

**Comment by chiggs** [comment]
Good work!

Note the @landscape-bot "new" errors reported above are simply places where you've touched the code and are actually false positives.

**Comment by chiggs** [comment]
Just a note on workflow.  It's a good idea to reference the issue in your commit messages.  For PEP8 we already have an issue so you could write something like this:

> #6 Numerous style improvements for PEP8

Also it would be preferable to rebase so that this all squashes to a single commit.

**Comment by mfkaptan** [comment]
> Also it would be preferable to rebase so that this all squashes to a single commit.

Done!

**Pull request merged**